### ISSUE TITLE
Tensor Board and Statistics

### ISSUE THREAD

**Issue by remzr7** [rbrigden/a3c#4]
We need to have training and testing analytics for the A3C implementation. Read the tensorflow documentation and figure out how we can hook up our model to the tensorboard and also record statistic tracking, including the following: - max V per episode
- avg reward per episode

**Comment by rbrigden** [comment]
So it would be really helpful for debugging purposes to track a few specific metrics for now.

- gradient norms
- ouput of value and policy networks
- objective functions (total, policy, and value)
- entropy
- layer outputs

**Comment by rbrigden** [comment]
Status?
